### PR TITLE
Correction to show real latest news

### PR DESCRIPTION
**Pull request by Aviorleking**
reversing the array to show the real latest(not the first)

### CODE REVIEW

File: modules/Forum/classes/Forum.php [hunk 1]
```diff
@@ -469,4 +469,4 @@ public function getPosts($tid = null){
 		}
 		return false;
 	}
-}
```
Comment (Aviorleking) [inline modules/Forum/classes/Forum.php]: wut

**Comment by Aviorleking** [comment]
you missed one word in the EnglishUK translation i added it

**Comment by samerton** [comment]
The order of news on the homepage should be correct without calling array_reverse:

Without array_reverse (latest news post is on top):
Before

With array_reverse (latest news post is now on the bottom):
After

**Comment by Aviorleking** [comment]
it happen the reverse for me without i have my latest at the bottom and with it it is at the top

i'm gonna give the proof later not on my computer

**Comment by Aviorleking** [comment]
with reverse : 
image

without : 

image

(first was the 1st april last was the 4th april)

i am using the latest of latest(cloning all changes)

**Comment by Derkades** [comment]
Could it be something with your database server? For example, in the code it may say `SELECT * FROM news ORDER BY date`. This is supposed to sort in ascending order, but may for some reason be descending for you

**Comment by samerton** [comment]
I've just been able to reproduce the issue, and I'll investigate at some point in the near future.

**Comment by Aviorleking** [comment]
Ok thanks

**Comment by samerton** [comment]
Thanks for bringing the issue up, I've just found the actual cause of the issue and have now solved it (https://github.com/NamelessMC/Nameless/commit/4711028eef58e9966089fa6288b5b2fed07814e8)

**Pull request closed without merging**